.travis.yml

```diff
@@ -22,6 +22,7 @@ env:
   - IMG=s3-loader/0.6.0
   - IMG=elasticsearch-loader/0.10.1
   - IMG=iglu-server/0.3.0
+  - IMG=piinguin-server/0.1.0
```

Comment:
There seem to be a mixture of 0.1.0 and 0.1.1 - should all refs be on the latest?

Reply:
yes